template/src/components/ColorHeader/ColorHeader.test.js

```diff
@@ -1,12 +1,15 @@
 import ColorHeader from './ColorHeader'
+import useColor from ''
```

Comment:
what is this import?

Reply:
@Nfinger, can you clarify what this import is for? This... doesn't appear right...

Reply:
Whoops, missed that one. Pushed up a change